code/datums/mutations/touch.dm

```diff
@@ -165,7 +165,7 @@
 		success = do_simple_heal(mendicant, hurtguy, heal_multiplier, pain_multiplier)
 
 	// Both types can be ignited (technically at least), so we can just do this here.
-	if(hurtguy.has_status_effect(/datum/status_effect/fire_handler/fire_stacks))
+	if(hurtguy.fire_stacks)
```

Comment:
You would want `fire_stacks > 0`, otherwise it would affect wetstacks (< 0).

Reply:
insane

```suggestion
	if(hurtguy.fire_stacks > 0)
```